---
 tools/building.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/building.py b/tools/building.py
index 91f5c6742..5dac9e3b5 100644
--- a/tools/building.py
+++ b/tools/building.py
@@ -271,8 +271,6 @@ def lld_flags_for_executable(external_symbols):
   # enabled.  So be explicit in all cases for now.
   if settings.STACK_FIRST:
     cmd.append('--stack-first')
-  else:
-    cmd.append('--no-stack-first')
 
   if not settings.RELOCATABLE:
     cmd.append('--table-base=%s' % settings.TABLE_BASE)
-- 
2.51.2
